src/parser/jobs/mnk/modules/RiddleOfFire.tsx

```diff
@@ -86,90 +121,140 @@ export default class RiddleOfFire extends Module {
 			this.stopAndSave()
 		}
 
-		// Aggregate GCDs under each RoF
-		const missedGcds = this.history
+		// The opener Riddle is special (10 GCDs with some FoF uptime), so keep track of it.
+		const openerRiddle = this.history[0]
```

Comment:
This is making an assumption the user is doing a proper opener.

Reply:
Right, but our ultimate goal is to not punish people who are doing proper openers.

Reply:
Now that I think about it, the core issue here is detecting RoF sequences that start at sub-GL4. Should be doable with the new gauge module stuff.